### PR TITLE
Seltza

### PR DESCRIPTION
**Pull request by alex-g-carpenter**
I think it's time we get some seltza in here? Wouldn't you all agree?

### CODE REVIEW

**Comment by colinrlly** [comment]
Hm suss

**Comment by HarperCScott** [comment]
This sucks so hard lmao.

**Comment by colinrlly** [comment]
Seltza is suss

**Comment by Chasemoudry** [comment]
"Ey ma, where's da seltza wahtuh?"

File: seltza/assignment3.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        new_length = branch_length  - shorten_by
+
+        t.left(angle)
+        build_tree(t, new_length, shorten_by, angle)
+
+        t.right(angle*2)
+        build_tree(t, new_length, shorten_by, angle)
+        
+        t.left(angle)
+        t.backward(branch_length)
```
Comment (colinrlly) [inline seltza/assignment3.py:30]: Lol nice pern code

File: seltza/assignment3.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Colin Reilly School of Brogramming Assignment 3
+
+s = turtle.getscreen()
+
+t = turtle.Turtle()
+
+# t.right(90)
+# t.forward(100)
+# t.left(90)
+# t.backward(100)
```
Comment (Chasemoudry) [inline seltza/assignment3.py:13]: Heard you like comments, so here's a comment for your comments.

File: seltza/assignment2.py [hunk 3]
```diff
@@ -0,0 +1 @@
+print('hollo wored')
```
Comment (HarperCScott) [inline seltza/assignment2.py:1]: Good spelling idiot go back to school.

File: seltza/assignment3.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+max_branch_length = 5
+
+
+def build_tree(t, branch_length, shorten_by, angle):
```
Comment (wakabayashin) [inline seltza/assignment3.py:18]: This is cheating

File: seltza/assignment3.py (hunk 2 above)
Comment (HarperCScott) [reply seltza/assignment3.py:13]: Shut up please.

File: seltza/assignment3.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    if branch_length > max_branch_length:
```
Comment (wakabayashin) [inline seltza/assignment3.py:19]: cheater

File: seltza/assignment3.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import turtle
+
+# Alex Carpenter
```
Comment (wakabayashin) [inline seltza/assignment3.py:3]: literally who

File: seltza/assignment3.py (hunk 2 above)
Comment (colinrlly) [reply seltza/assignment3.py:13]: We put a review in your comment so you can comment while you review

File: seltza/assignment4.py [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+t.forward(shaft_length+10)
+
+#for s in range(int(circle_sides/2)):
+#    t.forward(circle_side_len)
+#    t.right(360/circle_sides)
+
+#    t.right(90)
+
+
+turtle.done()
```
Comment (wakabayashin) [inline seltza/assignment4.py:47]: I like this code

File: seltza/assignment3.py (hunk 2 above)
Comment (HarperCScott) [reply seltza/assignment3.py:13]: All of you, shut up. Please.

Comment (HarperCScott) [reply seltza/assignment3.py:13]: I am begging you all to read a book.

**Comment by Chasemoudry** [comment]
it's ok now

File: seltza/assignment4.py (hunk 7 above)
Comment (alex-g-carpenter) [reply seltza/assignment4.py:47]: > 
> 
> This sucks so hard lmao.

WHAT ARE YOU GONNA DO ABOUT IT, HUH?

**Comment by alex-g-carpenter** [comment]
> 
> 
> This sucks so hard lmao.

WHAT ARE YOU GONNA DO ABOUT IT HUH?!?!

**Pull request merged**